src/test/librbd/test_librbd.cc

```diff
@@ -6843,41 +6880,48 @@ TEST_F(TestLibRBD, BreakLock)
 
   rbd_image_t image, blacklist_image;
   ASSERT_EQ(0, rbd_open(ioctx, name.c_str(), &image, NULL));
+  uint64_t features;
+  rbd_get_features(image, &features);
+  if (features && RBD_FEATURE_DIRTY_CACHE) {
```

Comment:
Nit: why does this test need to be bypassed?

Reply:
In this case, an image is opened twice. One gets the exclusive lock, and then the other breaks the exclusive lock. When breaking the exclusive lock, the operations in src/librbd/PreReleaseRequest.cc are not be executed.   
_As_ a result, the cache is not closed. So when it tried to accquire the exclusive lock, it failed to initialize the image cache.    
So this case can't be executed successfully.    
@dillaman I think we should disallow the lock break when RWL cache exists. How do you think?

Reply:
What happens if a client crashes? Breaking the exclusive lock is similar to force-killing a client. I guess I just want to ensure that you can recover from a client failure. If it's just because the pmem store is still opened by the blacklisted client, that's OK.

Reply:
In this test, it failed because the cache pool (pmem store) had been opened. If a client crashes and restarts, librbd will reopen the cache pool file, and reload the existed cached data.

Reply:
Can we just add a `REQUIRE(!is_rwl_cache_enabled())`-like method to avoid large-scale test changes just to avoid running the test? You would just need to check the config setting.

Reply:
Updated.